docs/configuration/modules/remote-service.md

```diff
@@ -31,6 +31,28 @@ to connect to the remote server. The default value of this property is `true`.
 The `port` property is an optional property that defines the connection port. By default, if SSL/TLS protocol
 is used, the default value of this property is `443`. Otherwise, it is `80`.
 
+## autoreconnect-enabled
+
+The `autoreconnect-enabled` property is an optional property that defines if auto reconnection to the server should be used when to disconnect. The default value of this property is `false`.
+
+## reconnection-initial-interval
+
+The `reconnection-initial-interval` property is an optional property that defines the reconnection initial interval. The default value of this property is `5`in seconds.
+
+## reconnection-interval-mutiplier
+
+The `reconnection-interval-mutiplier` property is an optional property that defines the reconnection interval mutiplier. The default value of this property is `2`.
+
+## reconnection-max-interval
+
+The `reconnection-max-interval` property is an optional property that defines the max interval between two reconnection. The default value of this property is `3600`in seconds.
```

Comment:
*reconnections

Specify that it is in seconds before mentioning the default property : 
"[...] that defines the max interval **in seconds** between two reconnections. Its default value is `3600`."

Reply:
It's done